modules/chat/src/Chat.ts

```diff
@@ -40,12 +41,17 @@ export default class Chat extends ManagedModule {
   }
 
   async onConfig() {
+    const config = await ConfigController.getInstance().config;
     if (!this.isRunning) {
       await this.registerSchemas();
       this.adminRouter = new AdminHandlers(this.grpcServer, this.grpcSdk);
       this.userRouter = new ChatRoutes(this.grpcServer, this.grpcSdk);
       this.isRunning = true;
     }
+    if (config.explicit_room_joins.send_email)
```

Comment:
This should be `if (config.explicit_room_joins.enabled && config.explicit_room_joins.send_email`.
Same with any other check that ends up awaiting email or sending invitation emails.

If we only have `config.explicit_room_joins.enabled === true` we merely create the invitations.
`send_email === true` is ignored while `config.explicit_room_joins.enabled === false`.